security/tls-1.3.md

```diff
@@ -51,27 +51,59 @@ TLS version is not configurable with flags.
 
 For Ironic, it should be noted that:
 
-- Ironic external endpoints are secured, while pod-internal traffic is not.
-  `httpd` (Apache) handles TLS termination.
-- IPA image serving from `httpd` port `6180` might be insecure due to PXE
-  limitations (to be verified)
-- Node image server is deployment specific. IPA client accessing the image
-  server is limited to TLS 1.2 (see Oslo below)
-
-Ports:
-
-- Port `5049`: HTTP (Ironic Inspector API)
+- IPA image serving from `httpd` port `6180` might be insecure due to BMC or
+  (i)PXE firmware limitations, the user has to enable TLS for both
+  iPXE and virtual media and in case of iPXE custom iPXE firmware building
+  is required. In case TLS is enabled for virtual media and iPXE, additional
+  TLS enabled ports will be opened in the form of `8083` and `8084`
```

Comment:
```suggestion
  (i)PXE firmware limitations, the user has to enable TLS for both
  iPXE and virtual media. In case of iPXE, custom iPXE firmware building
  is required. If TLS is enabled for virtual media and iPXE, additional
  TLS enabled ports will be opened in the form of `8083` and `8084`
```

Reply:
Thanks done.